packages/react-components/react-rating/src/components/RatingDisplay/useRatingDisplayStyles.styles.ts

```diff
@@ -21,18 +21,20 @@ const useRootClassName = makeResetStyles({
 
 const useLabelClassName = makeResetStyles({
   color: tokens.colorNeutralForeground1,
-  marginLeft: tokens.spacingHorizontalXXS,
+  marginLeft: tokens.spacingHorizontalXS,
```

Comment:
Someone pointed out logical properties on one of my PR's a little bit ago.... slightly hard to reason about, but makes handling RTL issues easier.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values

Reply:
This is a good callout. I will make a separate PR to update the general logic properties.